### DISCUSSION TITLE
Minor suggested changes to the CLI

### DISCUSSION THREAD

**Discussion by UnforeseenOcean** [Tyrrrz/DiscordChatExporter#1117] [General]
1. Changing the "filled" line character from a colored dash to a bigger character such as `#` so it looks like this:
```
Category / general #######################----------------- 50%
```
2. Two more digits below the decimal point for the percentage so I know if the program is frozen and downloading something it shouldn't or if it's working as intended
```
Category / general #######################----------------- 50.42%
```

This could be useful for people with bad eyesight or people with huge backups.

**Comment by Tyrrrz** [comment]
IIRC, the progress bar characters look different if you have Unicode support enabled in the terminal 🤔

**Reply by UnforeseenOcean** [reply]
I did, but for whatever reason the setting doesn't stick. Anyway, this applies for the fallback mode.

**Reply by Tyrrrz** [reply]
The detailed percentages are blocked by this for now: https://github.com/spectreconsole/spectre.console/issues/1277
